.vortex/tooling/src/helpers.php

```
@@ -0,0 +1,707 @@
+
+  /**
+   * Exit script with given code.
+   *
+   * Wrapper around exit() to allow mocking in tests since exit() cannot be
+   * directly mocked despite being a function in PHP 8.4+.
+   *
+   * @param int $code
+   *   Exit code (0 for success, non-zero for error).
+   */
+  function quit(int $code = 0): void {
+    exit($code);
```

PHPMD (2.15.0)

[error] The function quit() contains an exit expression. (undefined)

(ExitExpression)